src/mpr/utility/getParams.ts

```diff
@@ -0,0 +1,198 @@
+/**
+ * EC-LAB TECHNIQUE PARAMS
+ */
+type Technique = string;
+type ArraySS = [string, string][];
+export function getParams(id: number): [Technique, ArraySS] {
```

Comment:
I'm not fan of returning an array. Could you rather return an object
{technique: abc, parameters: bcd} or something like that ?
There may be some other refactor like this one.

Reply:
At the moment that one is still a tuple/array. Will try to improve it at some point. Yes I don't like either.